llvm/test/SYCLLowerIR/ConditionalCallOnDevice/conditional-call-on-device-2.ll

```diff
@@ -0,0 +1,55 @@
+; RUN: opt -passes=sycl-conditional-call-on-device < %s -S | FileCheck %s
```

Comment:
Unique prefix should be configurable through options so that we are able to test that it is actually taken into account by the pass. See [CommandLine](https://llvm.org/docs/CommandLine.html), `llvm/test/SYCLLowerIR/ESIMD/acc_ptr.ll` as an example of a test and `llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp` as an example of a pass